src/test/java/org/dataloader/DataLoaderBatchLoaderEnvironmentTest.java

```diff
@@ -66,10 +67,14 @@ public void key_contexts_are_passed_to_batch_loader_function() {
         loader.load("A", "aCtx");
         loader.load("B", "bCtx");
         loader.loadMany(asList("C", "D"), asList("cCtx", "dCtx"));
+        Map<String, String> keysAndContexts = new LinkedHashMap<>();
+        keysAndContexts.put("E", "eCtx");
+        keysAndContexts.put("F", "fCtx");
+        loader.loadMany(keysAndContexts);
```

Comment:
Used `LinkedHashMap` because it is `null safe` and retains ordering (so the test(s) should not be flaky).